plugin-config.testing.js

```diff
@@ -1,10 +1,16 @@
 const { PluginScope } = require("./Enums.js");
+const bug_options = require("./assets_plugins/json/bug-report-questions.js");
 
 const config = [
   {
     id: "plugin-bug-report",
     scope: PluginScope.PAGE,
-    js_modules: ["https://vjspranav.github.io/vleads-bug-report/client/app.js"],
+    js_modules: [
+      "https://vjspranav.github.io/vleads-bug-report/client/src/bug-report.js",
+    ],
+    attributes: {
+      bug_options: JSON.stringify(bug_options),
```

Comment:
try to find better(more generic) names please.

Reply:
for inside attributes, I was planning, to just loop over all attributes, and assign them to webcomponent, this allows us to define attributes in json if we need to, mainly for questions